debug_toolbar/static/debug_toolbar/js/toolbar.js

```diff
@@ -55,6 +57,7 @@ const djdt = {
                                 detail: { panelId: panelId },
                             })
                         );
+
```

Comment:
```suggestion
```
Removing unnecessary whitespace change. (you can ignore these, I'll batch the changes together)